osu.Game/Overlays/OnScreenDisplay.cs

```diff
@@ -160,6 +160,10 @@ public void StopTracking(object source, ITrackableConfigManager configManager)
         {
             if (configManager == null) throw new ArgumentNullException(nameof(configManager));
 
+            var trackedSettings = configManager.CreateTrackedSettings();
+            if (trackedSettings == null)
+                return;
```

Comment:
It's throwing on the line below, right? Perhaps it shouldn't throw and instead silently early-return.

Reply:
also I'd prefer `CreateTrackedSettings` wasn't called a second time here *just* for this check.